### PR TITLE
Add link to supported networking addons in "Creating a cluster with kubeadm" page

### PR DESCRIPTION
**Pull request by steve-hardman**
Using one of the suggested solution in https://github.com/kubernetes/website/issues/45007#issuecomment-1925442177 in this PR to fix issue https://github.com/kubernetes/website/issues/45007

~Closes&nbsp;https://github.com/kubernetes/website/issues/45007~

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -333,8 +333,10 @@ support [Network Policy](/docs/concepts/services-networking/network-policies/).
 See a list of add-ons that implement the
 [Kubernetes networking model](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-network-model).
 
-You can install a Pod network add-on with the following command on the
-control-plane node or a node that has the kubeconfig credentials:
+
+Please refer to the Installing Addons page for a non-exhaustive list of networking 
+addons supported by Kubernetes You can install a Pod network add-on with the following 
+command on the control-plane node or a node that has the kubeconfig credentials:
```
Comment (dipesh-rawat) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:339]: The "Installing Addons" text seems to be missing the inline link to the page, and there's also a minor punctuation oversight in the new line.

```suggestion
Please refer to the [Installing Addons](/docs/concepts/cluster-administration/addons/#networking-and-network-policy) page for a non-exhaustive list of networking 
addons supported by Kubernetes. You can install a Pod network add-on with the following 
command on the control-plane node or a node that has the kubeconfig credentials:
```

**Comment by dipesh-rawat** [comment]
/retitle Add link to supported networking addons in "Creating a cluster with kubeadm" page

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md (hunk 1 above)
Comment (steve-hardman) [reply content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:339]: Correction added as part tof latest commit. Thanks!

**Review by sftim — COMMENTED** [review]
I don't want to approve this until the PR is no longer marked as “Closes #45007”; closing that issue early would be unhelpful.

I think **this is a useful interim fix** and that we have scope to help readers even further. https://k8s.io/docs/concepts/cluster-administration/addons/#networking-and-network-policy is a reference page (list of addons), with a title that sounds like a task (but doesn't explain how to set any of them up, in the concepts section.

I hope it's obvious that there is scope to improve things here.

**Review by neolit123 — COMMENTED** [review]
/lgtm
/sig cluster-lifecycle
/triage accepted

thanks!

**Comment by sftim** [comment]
I edited the PR description so that this does not close #45007 

/approve

**Pull request merged**